requirements-chrouter.in

```diff
@@ -0,0 +1,14 @@
+-e git+https://github.com/assembl/sockjs-tornado.git@conflicting_localhost#egg=sockjs_tornado
```

Comment:
what is chrouter? why is it separated from the rest?

Reply:
probably document this answer somewhere

Reply:
changes_router... sorry for the abbreviation. added a comment. Separate because if it's put in its own docker component, the requirements are completely different from main assembl.

Reply:
I just added even more explanation from what I understood after this comment and a bit more research, because it was still not clear to me.